Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
+use EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SystemInfoDumpCommand extends ContainerAwareCommand
 {
     /**
+     * System info collector registry.
+     *
+     * @var \EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry
+     */
+    private $registry;
+
+    public function __construct(SystemInfoCollectorRegistry $registry)
+    {
+        $this->registry = $registry;
+
+        parent::__construct();
+    }
+
+    /**
      * Define command and input options.
      */
     protected function configure()
     {
         $this
             ->setName('ez-support-tools:dump-info')
-            ->setDescription('Collects system information and dumps it.')
+            ->setDescription(<<<'EOD'
```

Comment:
The description should be one line, or we get this when running `app/console`:

```
 ez-support-tools
  ez-support-tools:dump-info            Collects system information and dumps it.

By default it dumps information from all available information collectors.
You can specify one or more collectors as arguments, e.g. 'php database hardware'.
To get a list if available collectors, use '--list-info-collectors'
 ezplatform
```

This should do:
```
            ->setDescription('Collects system information and dumps it.')
            ->setHelp(<<<EOH
By default it dumps information from all available information collectors.
You can specify one or more collectors as arguments, e.g. 'php database hardware'.
To get a list if available collectors, use '--list-info-collectors'
EOH
            )
```